### PR TITLE
Refferer build

### PR DESCRIPTION
**Pull request by Mi-Lan**
- Includes changes from reefferer pr along with build so that it can be tested on the app https://github.com/ImpeccableHQ/swapr-sdk/pull/280

### CODE REVIEW

File: src/entities/trades/0x/0xTrade.ts [hunk 1]
```diff
@@ -123,7 +124,7 @@ export class ZeroXTrade extends TradeWithSwapTransaction {
         }&slippagePercentage=${new Percent(
           maximumSlippage.numerator,
           JSBI.multiply(maximumSlippage.denominator, JSBI.BigInt(100))
-        ).toFixed(4)}`
+        ).toFixed(4)}&feeRecipient=${REFFERER_ADDRESS_CHAIN_MAPPING[chainId]}&buyTokenPercentageFee=${fee}`
```
Comment (berteotti) [inline src/entities/trades/0x/0xTrade.ts:127]: Should we had a fallback in case `REFFERER_ADDRESS_CHAIN_MAPPING` returns `undefined`?

**Review by berteotti — COMMENTED** [review]
Looks good to me, just little improvements. Good job!

File: src/entities/trades/OneInch/api.ts [hunk 2]
```diff
@@ -13,7 +14,16 @@ interface ApiRequestUrlParams {
...
 
+//0.1% fee here is link to api https://docs.1inch.io/docs/aggregation-protocol/api/swap-params
+
+const fee = '0' //MIN-> 0 MAX-> 3
+
 export function generateApiRequestUrl({ methodName, queryParams, chainId }: ApiRequestUrlParams) {
+  queryParams.referrerAddress = REFFERER_ADDRESS_CHAIN_MAPPING[chainId] ?? ''
+  if (REFFERER_ADDRESS_CHAIN_MAPPING[chainId]) {
+    queryParams.fee = fee
+  }
```
[the remark this answers is not in the document]
Comment (Mi-Lan) [reply src/entities/trades/OneInch/api.ts:25]: good call!

**Review by berteotti — COMMENTED** [review]
Code looks good, don't forget to remove dist folder before merging

**Pull request closed without merging**